### PR TITLE
Documentation updates for Logi Circle public API refactor and config flow

### PR DESCRIPTION
**Pull request by evanjd**
**Description:**

The Logi Circle integration has been updated to use Logitech's public API and contains many breaking changes. Refer to Home Assistant PR for more detail.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#20624

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Name of the files are not correct.
logi_circle.camera.markdown -> camera.logi_circle.markdown
logi_circle.sensor.markdown -> sensor.logi_circle.markdown

And no support for a Binary Sensor? I see here in the old PR a file about that: https://github.com/home-assistant/home-assistant.io/pull/8197/files

**Comment by evanjd** [comment]
> Name of the files are not correct.
logi_circle.camera.markdown -> camera.logi_circle.markdown
logi_circle.sensor.markdown -> sensor.logi_circle.markdown

@klaasnicolaas I was following the example used by other components with embedded platforms.

For example, the smartthings PR that was just merged:
https://github.com/andrewsayre/home-assistant.io/blob/next/source/_components/smartthings.switch.markdown
And the related conversation where this change was requested: https://github.com/home-assistant/home-assistant/pull/20148#discussion_r248455136

Let me know if I'm misinterpreting.

> And no support for a Binary Sensor? I see here in the old PR a file about that: https://github.com/home-assistant/home-assistant.io/pull/8197/files

I was asked to remove it as my first PR was too big. I'll be raising another PR with the binary sensor (and other changes) as soon as this cut-down PR is merged.

**Comment by klaasnicolaas** [comment]
That of smartthings is a mistake that has crept into it.
sensor.(component).markdown is the usual order we use.

**Comment by evanjd** [comment]
I'm sorry to be a pain, but I want to double check that you saw the conversation I linked where @balloob says to use `(component).(platform).markdown` for components with embedded platforms, at least in smartthings case.

The other problem with renaming as you suggested is that the generated source link is wrong.

Here's how it looks now (correct):
screen shot 2019-02-01 at 12 30 51 am

With the change you requested (incorrect):
screen shot 2019-02-01 at 12 34 13 am

**Comment by klaasnicolaas** [comment]
Think it's best if @balloob looks at this PR.

**Comment by balloob** [comment]
Names should follow the source code. So `logi_circle/sensor.py` is the right one.

**Comment by klaasnicolaas** [comment]
> Names should follow the source code. So `logi_circle/sensor.py` is the right one.

So..`logi_circle.sensor.markdown`?

**Comment by balloob** [comment]
Yes

**Comment by frenck** [comment]
Just spoke with @balloob, currently, we use: `(platform).(component).markdown`.

Our code base did a "flip-flop", but for our documentation, we are not ready for that yet.
We are currently merging components with smaller platform pages into a single page to reduce the number of pages we have.

A "flip-flop" done as in the code base, might happen in the future but is not decided on yet.

So, please use `(platform).(component).markdown` OR just create a single component page (which is currently preferred because of the current docs changes in progress).

Sorry for the confusion that this may have caused.

**Comment by evanjd** [comment]
No problem, thanks for clarifying @frenck and sorry for the initial hassle @klaasnicolaas.

I've rebased the PR and merged the camera and sensor docs into a single logi_circle component page.

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
✅ I think it can be merged as soon the parent PR got merged.

**Comment by klaasnicolaas** [comment]
⚠️ We have some merge conflicts 😉

**Comment by evanjd** [comment]
@klaasnicolaas Fixed, sorry for the delay!

File: source/_components/logi_circle.markdown [hunk 1]
```diff
@@ -8,12 +8,36 @@ comments: false
...
-ha_category: Camera
+ha_category:
+  - Camera
+  - Sensor
 ha_release: 0.79
 ha_iot_class: Cloud Polling
+redirect_from:
+  - /components/camera.logi_circle/
+  - /components/sensor.logi_circle/
+  - /components/logi_circle.camera/
```
Comment (MartinHjelmare) [inline source/_components/logi_circle.markdown]: Have these pages existed previously? The address isn't correct.

```
/components/logi_circle.camera/
/components/logi_circle.sensor/
```

Comment (evanjd) [reply source/_components/logi_circle.markdown]: They didn't. I only added them to match the import path of the component, but happy to get rid of it if it's not appropriate.

Comment (klaasnicolaas) [reply source/_components/logi_circle.markdown]: If you remove them then we can perform the merge 😉

Comment (evanjd) [reply source/_components/logi_circle.markdown]: @klaasnicolaas Done.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/logi_circle.markdown [hunk 2]
```diff
@@ -22,19 +44,122 @@ To integrate cameras linked with your [Logi Circle](https://circle.logi.com/) ac
...
+
+The `logi_circle` camera platform allows you to view still frames from your [Logi Circle](https://circle.logi.com/) camera's live stream in Home Assistant.
+
+Logi Circle cameras support the `camera.turn_on` and `camera.turn_off` services. This will set the streaming mode property of your camera accordingly, controlling whether the live stream is available and activity recordings are captured.
+
+### {% linkable_title Sensor %}
+
+The `logi_circle` sensor platform lets you monitor sensors connected to your [Logi Circle](https://circle.logi.com) cameras in Home Assistant.
+
+To customize which sensors are setup, you can extend the [Logi Circle component](/components/logi_circle/) configuration in your `configuration.yaml` file with the following settings:
```
Comment (klaasnicolaas) [inline source/_components/logi_circle.markdown]:
```suggestion
To customize which sensors are setup, you can extend the Logi Circle component configuration in your `configuration.yaml` file with the following settings:
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Time to merge 😄

**Pull request merged**